### DISCUSSION TITLE
Keep alive 24/7 not running of a PC

### DISCUSSION THREAD

**Discussion by HysteriaMC** [ORelio/Minecraft-Console-Client#2024] [Q&A]
Hey,

So I want to keep my account running 24/7 ideally, is it possible to add MCC to some kind of server or something like that, to keep it running 24/7, without using a computer?

EDIT: I've worked out how to do this thanks to ReinforceZwei, but I want to change some settings, and using ubuntu, how can I do that? I want it to save my login details, etc
Thanks!

**Comment by ReinforceZwei** [comment]
You can rent a VPS and run your client on it.

**Reply by HysteriaMC** [reply]
> You can rent a VPS and run your client on it.

Thanks! Do you have any suggestions?
